packaging/ios/build.sh

```diff
@@ -48,7 +48,7 @@ popd
 # prepare build dir
 mkdir -p build-ios
 
-IOS_QT=${TOP}/Qt
+IOS_QT=~/Qt
```

Comment:
Funny story... the reason for this is that I used to build macOS and iOS from the same tree - and at times had different Qt versions. I no longer do that, so this is fine with me.